### DISCUSSION TITLE
Transclude part of drawing into another drawing

### DISCUSSION THREAD

**Discussion by KimAndersson** [zsviczian/obsidian-excalidraw-plugin#2100] [Ideas]
My dream for taking visual thinking to the next step with Obsidian Excalidraw would be to be able to translude a part of an Obsidan Excalidraw drawing into another Obsidian Excalidraw drawing.

Often I like to work with my visuals on many different levels, however, on the top level I do not want to show the entire drawing of the level below but instead a summary (a choosen part) of the drawing below.

See attached files for an example use case.
On the top level I only want to see the "Process step 2" process step (not 2.1, 2.2 and 2.3), however I want to include more visuals in the drawing for "Process step 2" which I can see and work with when I go into that drawing.

I hope there are more persons having the same need as I do, and I hope Zsolt will consider adding this feature in the future.

![Screenshot 2024-11-04 201003](https://github.com/user-attachments/assets/f02166f3-9caf-4227-b90f-67576bc5d8dc)
![Screenshot 2024-11-04 201135](https://github.com/user-attachments/assets/5d0b61d4-3e6c-4f92-9653-b9f0c079723e)

**Comment by LeonelRFF** [comment]
I think it would be useful to install the script "Add Link to New Page and Open" which allows you to create a drawing and link it to the selected object.
